docs/reST/ref/cursors.rst

```diff
@@ -199,6 +207,14 @@ The following strings can be converted into cursor bitmaps with
 
    Width and height must be a multiple of 8, and the mask arrays must be the 
    correct size for the given width and height. Otherwise an exception is raised.
+   
+   .. method::copy
+      | :sl:`copy the current cursor`
+      | :sg:`copy() -> Cursor`
+      
+      Returns a new Cursor object with the same data and hotspot as the original.
+   .. ## pygame.cursors.Cursor.copy ##
+   
```

Comment:
this doesn't show up like at all when I generate the docs locally. Maybe it's because of the spacing? You're going to have to generate the docs locally, I think. It also updates some header files with the docstrings, so that change should also be committed. You can try `setup.py docs`, and if that doesn't work you could rebase this branch against the current main, where it should be easier to generate them. If you're newer to git (like me), the easiest way to do that would just be to close this PR and make a new one with the changes off of current main.

Reply:
I actually figured it out, a spacing mistake